cmd/common-main.go

```diff
@@ -123,3 +123,27 @@ func handleCommonEnvVars() {
 		globalIsEnvDomainName = true
 	}
 }
+
+func loadCerts() {
```

Comment:
As gateway and server have different logic of handling TLS, this function is not required i.e. handle cases individually in gateway and server.

Reply:
Okay